libs/sdk-ui-dashboard/src/dashboard/Dashboard.tsx

```diff
@@ -41,8 +42,17 @@ export interface IDashboardProps {
      */
     dashboardRef?: ObjRef;
 
+    /**
+     * Configuration that modifies to dashboard features, capabilities and behavior.
```

Comment:
Nitpick: strange wording: modifies to dashboard

Reply:
fixed